atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -211,11 +209,7 @@ def read_configs(self):
         if not data and not self.all_compose_ids:
             raise SkipResolveComposesPlugin('"compose" config not set and compose_ids not given')
 
-        workdir = self.workflow.source.get_build_file_path()[1]
-        file_path = os.path.join(workdir, REPO_CONTENT_SETS_CONFIG)
-        pulp_data = None
-        if os.path.exists(file_path):
-            pulp_data = read_yaml_from_file_path(file_path, 'schemas/content_sets.json') or {}
+        pulp_data = util.read_content_sets(self.workflow)
```

Comment:
missing : or {}
OR you could also return those default values []/{} in those util reading methods

Reply:
actually it doesn't matter in this case that much, but still

it would be good to provide those default values in reading methods, and then you can remove those fallbacks to []/{}